fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -89,6 +89,17 @@ object ItemKicker {
     case FreeHtmlKickerWithLink(body, _) => Some(body)
     case _ => None
   }
+
+  /** Return the canonical display text for a kicker */
+  def kickerDisplayText(itemKicker: ItemKicker): Option[String] = itemKicker match {
```

Comment:
Why have this function at all? Wouldn't it make sense to just use that one function for all of the types, instead of doing half here and half over there? It makes it hard for someone using the client to know which of these two functions is the right one to use.